terra-jupyter-aou/Dockerfile

```diff
@@ -30,10 +30,16 @@ ENV PIP_USER=false
 ENV PYTHONPATH $PYTHONPATH:/usr/lib/spark/python
 
 RUN pip3 install \
+      igv-jupyter \
       nbstripout \
       papermill \
       "git+https://github.com/all-of-us/workbench-snippets.git#egg=terra_widgets&subdirectory=py"
 
+RUN jupyter nbextension enable --py igv
+
+# For gcloud alpha storage support.
+RUN pip3 install google-crc32c --target /usr/lib/google-cloud-sdk/lib/third_party
```

Comment:
FYI: `gcloud alpha storage cp` requires this package for large files. This can perform 4-5x better than gsutil for singular large file download: https://cloud.google.com/blog/products/storage-data-transfer/new-gcloud-storage-enables-super-fast-data-transfers

I didn't test this in the base image, it's possible it already works there, or else this could be integrated upstream if desired.

Reply:
Would be nice to move it to base image. But it's fine if you just wanna get sth out for AOU image asap

Reply:
Done - PTAL

Reply:
this can be removed since you added it in base?

Reply:
Thanks, good catch